src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
-body {
+@import "./abstracts/vars";
+@import "./abstracts/mixins";
+@import "./abstracts/functions";
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  overflow-x: hidden;
```

Comment:
Is it necessary?

Reply:
I agree, in this case it's not. However I am adding this rule in every project I am working on. This and a bunch of others like mixins for antialiasing for example.